modules/managing-memcached-operator-using-olm.adoc

```diff
@@ -46,27 +50,43 @@ for more information on manually defining a manifest file.
 
 . *Deploy the Operator.*
 
-.. Deploy an Operator by applying the Operator’s manifest to the desired namespace
-in the cluster:
+.. Create an OperatorGroup that specifies the namespaces that the Operator will
+be targeting. Create the following OperatorGroup in the namespace where you will
+create the CSV. In this example, the `default` namespace is used:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha2
+kind: OperatorGroup
+metadata:
+  name: memcached-operator-group
+  namespace: default
+  spec:
+    targetNamespaces:
+    - default
+----
+
+.. Apply the Operator's CSV manifest to the desired namespace in the cluster:
 +
 ----
 $ curl -Lo memcachedoperator.0.0.1.csv.yaml https://raw.githubusercontent.com/operator-framework/getting-started/master/memcachedoperator.0.0.1.csv.yaml
 $ kubectl apply -f memcachedoperator.0.0.1.csv.yaml
 $ kubectl get ClusterServiceVersion memcachedoperator.v0.0.1 -o json | jq '.status'
 ----
 
-.. After applying this manifest, nothing has happened yet, because the cluster does
-not meet the requirements specified in our manifest. Create the RBAC rules and
-`CustomResourceDefinition` for the Memcached type managed by the Operator:
+.. After applying this manifest, nothing has happened yet, because the cluster
+does not meet the requirements specified in our manifest. Create the role, role
+binding, and service account to grant resource permissions to the Operator, and
+the Custom Resource Definition (CRD) to define the Memcached type managed by the
+Operator:
 +
 ----
-$ kubectl apply -f deploy/rbac.yaml
-$ kubectl apply -f deploy/crd.yaml
+$ kubectl create -f deploy/crds/cache_v1alpha1_memcached_crd.yaml
```

Comment:
Will you double-check that we have all of these files at this point? I don't see where we created or generated them.

Reply:
@kalexand-rh Thanks for the review. And yes, you'll have these files when you build the Memcached Operator, which is a prerequisite to this procedure.

Reply:
Though I did have a NOTE here that gave info about this, but removed it because the files changed. I'll look into adding something to clarify this.